class1/README.md

```diff
@@ -1 +1,131 @@
 #Class 1
+
+##Goals: 
```

Comment:
I think if we rephrase it from "students" to "you", it can be helpful for students to understand what the expectations are for each class. i.e. how they should spend their time and attention. Keep them from worrying about 'the best way' to do something and instead focus on js fundamentals at this point, for example.